src/scripts/filtering.css

```diff
@@ -1 +1,5 @@
-.renderIfVisible:has([data-filtering="hidden"]) { display: none; }
\ No newline at end of file
+.renderIfVisible:has([data-filtering="hidden"]) { display: none; }
+
+.-chutils-remove-shares--hide-shares .-chutils-remove-shares--is-share {
```

Comment:
is there a reason the same hidden attribute wouldn't also work for hiding shares?

Reply:
when the checkbox is clicked, the .-chutils-remove-shares--hide-shares class is toggled, which allows shares to be toggled on and off without mutating the DOM again

Reply:
oops, my bad, disregard